extensions/interactions/ItemSelectionInput/protractor.js

```diff
@@ -0,0 +1,159 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end testing utilities for the Item Selection
+ * interaction.
+ */
+
+var forms = require('../../../core/tests/protractor_utils/forms.js');
+var objects = require('../../objects/protractor.js');
+var waitFor = require('../../../core/tests/protractor_utils/waitFor.js');
+
+// The members of richTextInstructionsArray are functions, one for each option,
+// which will each be passed a 'handler' that they can use to edit the
+// rich-text area of the option, for example by
+//   handler.appendUnderlineText('emphasised');
+var customizeInteraction = function(
+    elem, richTextInstructionsArray, maxSelectionAllowed) {
+  objects.IntEditor(
+    elem.all(by.tagName('schema-based-int-editor')).last()
```

Comment:
I'm not 100% sure, but I think it would be difficult to use a unique HTML class since both integer input boxes are generated by the same directive. I chose to use `.last()` to select the second input box (which determines the max selections allowed) because I saw that's how a few other tests (e.g. the Video test) approached it when looking through the codebase to figure out how best to do it!

Reply:
Maybe you could identify the correct input field using the prompt text?

![Screen Shot 2020-04-09 at 13 14 36](https://user-images.githubusercontent.com/19878639/78922184-225c1600-7a64-11ea-991c-582085604243.png)

Reply:
Done!